### ISSUE TITLE
permanently hide mac menu bar

### ISSUE THREAD

**Issue by jaden-falls** [koekeishiya/yabai#1868]
is there anyway to change Mac system files to permanently hide the menu bar? i don't mind if it messes with my os

**Comment by srgtmshnk** [comment]
> is there anyway to change Mac system files to permanently hide the menu bar? i don't mind if it messes with my os

i dont think you even want to hide it, apple's applications make their toolbar in this menu (for example safari). so i think the only solution here is to hide it unless you hover on the top side of the screen

**Comment by Khodaparastan** [comment]
you can Only make it auto hide 
I personally have turned the auto hide option on and use [sketchybar](https://github.com/FelixKratz/SketchyBar) as status bar

**Comment by koekeishiya** [comment]
`yabai -m config menubar_opacity 0.0` in yabai v6.0.12 and newer.
